CRM/Upgrade/Incremental/php/FiveZero.php

```diff
@@ -106,6 +108,40 @@ public function upgrade_5_0_1($rev) {
     }
   }
 
+  /**
+   * Upgrade function for version 5.0.2. This version adds the category column
+   * to case types. The category is an option value that can be Workflow or
+   * Vacancy.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_0_2($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
+
+    CRM_Core_BAO_OptionGroup::ensureOptionGroupExists(array(
+      'name' => 'case_type_category',
+      'title' => ts('Case Type Category'),
+      'is_reserved' => 1,
+    ));
+
+    CRM_Core_PseudoConstant::flush();
+
+    $options = array(
+      array('name' => 'WORKFLOW', 'label' => ts('Workflow')),
+      array('name' => 'VACANCY', 'label' => ts('Vacancy'))
+    );
+
+    foreach ($options as $option) {
+      CRM_Core_BAO_OptionValue::ensureOptionValueExists(array(
+        'option_group_id' => 'case_type_category',
+        'name' => $option['name'],
+        'label' => $option['label'],
```

Comment:
What about the value property of the option value? It will be set as an integer by civi though.

Reply:
It's set automatically, should I set it manually? It seems civi prefers to use this value as an INT